migration_tools.drush.inc

```diff
@@ -248,6 +263,51 @@ function drush_migration_tools_mt_migrate_file_folders($organization) {
 }
 
 /**
+ * Drush command callback for migration-tools-html-file-list.
+ *
+ */
+function drush_migration_tools_html_file_list($directory) {
+  // Set up options
+  $format = drush_get_option('format');
+  $outfile = drush_get_option('outfile');
+  $mask = '/\.htm/';
+
+  switch ($format) {
+    case "csv":
+      $delimiter = ", ";
+      $newline = "\n";
+      break;
+    default:
+      $delimiter = "\t";
+      $newline = "\n";
+  }
+
+  // Fetch file list
+  $options = array();
+  $files = file_scan_directory($directory, $mask, $options);
+
+  // Generate output
+  $output = "Path" . $delimiter . "File Name" . $delimiter . $newline;
+  foreach ($files as $file) {
+    // We only want the path to the file, exclusive of the file name, so we drop the last part of the uri
+    $fullpath = explode("/", $file->uri, -1);
+    $output .= implode("/", $fullpath) . $delimiter;
+
+    $output .= $file->filename . $delimiter;
+    $output .= $newline;
+  }
+
```

Comment:
Oh.  Sorry I didn't catch that..... we don't want to break them into separate columns for directory and file.   We want them joined together because that is the unique id that we use to migrate or exclude each file.   If we go to all the trouble to split them now, we have to go through the trouble to join them later.  Less trouble more :)